blocks/library/quote/index.js

```diff
@@ -238,7 +240,7 @@ registerBlockType( 'core/quote', {
 
 		return (
 			<blockquote
-				className={ `blocks-quote-style-${ style }` }
+				className={ style === 2 ? 'is-large' : '' }
```

Comment:
I think we could use `default` and `large` for the attribute itself? Or `undefined` and `large` maybe.

Reply:
mmm, this means the value of the attribute can change between two versions, I'm going to introduce a `migrate` function to handle this

Reply:
Mmm, can it be handled in the attribute itself as a fallback?

Reply:
We might want to sit on it a bit otherwise.

Reply:
No, we can't :(. Adding it is not so complex though 5dd81ce757c33f7c6d9e3d8a9db9166cae8068a3

Reply:
Let's extract that commit into a separate PR so we can discuss the migration of values in isolation.